tests/integration/cli_test.py

```diff
@@ -5,9 +5,9 @@
 import sys
 from operator import attrgetter
 
-from mock import patch
 from six import StringIO
 
+from .. import mock
```

Comment:
I tried this out again, and I remember why I did it this way.  In py2.7 we're getting mock from the third-party package, but in py3.4 we have to use the mock that is included with python (because the third-party package stopped working for some patches).

If I try to use  `from tests.mock import patch` or `from ..mock import patch` I get  `ImportError: No module named mock`

I'm not sure if there's a way to make it work